src/devtools/client/debugger/src/components/QuickOpenModal.js

```diff
@@ -172,7 +177,7 @@ export class QuickOpenModal extends Component {
     }
 
     return this.searchSources(query);
-  }, updateResultsThrottle);
+  }, this.getDebounceMs());
```

Comment:
I'd be surprised if this works if the source count could change.

There should be a way to set updateResults whenever the `sourceCount` updates in a functional components way that would be a useEffect...perhaps here it is in `componentDidUpdate` and `this.updateResults = () => {}` this should not be hard but it is silly

Reply:
Yep. Fixed it so that we update the callback with a new debounced function with the right `ms` whenever the `sourceCount` changes.